src/main/java/com/google/swarm/tokenization/common/FileReaderSplitDoFn.java

```diff
@@ -97,9 +105,9 @@ public OffsetRangeTracker newTracker(@Restriction OffsetRange range) {
     return new OffsetRangeTracker(new OffsetRange(range.getFrom(), range.getTo()));
   }
 
-  private static SeekableByteChannel getReader(ReadableFile eventFile) throws IOException {
+  private static SeekableByteChannel getReader(ReadableFile eventFile) throws IOException, FileNotFoundException {
     SeekableByteChannel channel = null;
-    LOG.info("event File Channel {}",eventFile.getMetadata().resourceId().getFilename());
+    LOG.info("File Read Transform:ReadFile: event File Channel {}",eventFile.getMetadata().resourceId().getFilename());
```

Comment:
catch block in process element should output this. Just take out log.info and print out the error. Same as here:https://github.com/GoogleCloudPlatform/dlp-dataflow-deidentification/blob/2c9bbd178ccaaa4422f89ba49fd0e8b4a0e4f26b/src/main/java/com/google/swarm/tokenization/S3Import.java#L345

Reply:
Okay.